src/sentry/integrations/vercel/integration.py

```diff
@@ -48,6 +49,8 @@
     aspects={},
 )
 
+internal_integration_overview = "This internal integration was auto-generated during the installation process of your Vercel integration. It is needed to provide the token used to create a release. If this integration is deleted, your Vercel integration will stop working!"
```

Comment:
```suggestion
internal_integration_overview = (
    "This internal integration was auto-generated during the installation process of your Vercel"
    " integration. It is needed to provide the token used to create a release. If this integration is "
    "deleted, your Vercel integration will stop working!"
 )
```

python will glue all these string fragments together.

Reply:
@markstory haha...so obvious in retrospect. Thanks!